go.mod

```diff
@@ -74,6 +75,7 @@ require (
 	github.com/nats-io/stan.go v0.4.5
 	github.com/ncw/directio v1.0.5
 	github.com/nsqio/go-nsq v1.0.7
+	github.com/pierrec/lz4 v2.0.5+incompatible
```

Comment:
Quite old version. I contributed some speedups that were released in v2.2.4 which significantly improves compression speed.

Reply:
Yeah this was auto addition by `go mod` - will have to check the right version released.